_search-plugins/ubi/ubi-dashboard-tutorial.md

```diff
@@ -0,0 +1,98 @@
+![Word Cloud]({{site.url}}{{site.baseurl}}/images/ubi/tag_cloud1.png "Word Cloud")
+
+The underlying queries can be found under [SQL trending queries]({{site.url}}{{site.baseurl}}/search-plugins/ubi/sql-queries/#trending-queries).
+{: .note} 
+
+
+However, this visualization is not exactly what you're looking for. The `message` field is updated on every event---not only query/search events---and can be used in any way the client developer decides to use it. As a result, it can contain error messages, debug messages, click information, and other data.
+To view only search terms on query events, you need to add a filter to your visualization. Since the developer provided a `message_type` of `QUERY` for each search event, you can filter on that message type to isolate the specific users' searches. To do this, select **Add filter** and then select **QUERY** in the **Edit filter** panel, as shown in the following image. 
+![Word Cloud]({{site.url}}{{site.baseurl}}/images/ubi/tag_cloud2.png "Word Cloud")
+
+You should now have two visualizations on your dashboard (the pie chart and the tag cloud), as shown in the following image.
+![UBI Dashboard]({{site.url}}{{site.baseurl}}/images/ubi/dashboard2.png "UBI Dashboard")
+
+## 5. Add a histogram of item clicks
+
+Now you'll add a histogram visualization to your dashboard, similar to the previous step. In the **New Visualization** window, select **Vertical Bar**. Then select the index pattern you created in Step 2. 
+
+The data field you want to examine is `event_attributes.position.ordinal`. This field contains the position of the item in a list that the user selected. For the histogram visualization, the x-axis will be the ordinal number of the selected item (n). The y-axis will be the number of times that the nth item was clicked, as shown in the following image. 
+
+![Vertical Bar Chart]({{site.url}}{{site.baseurl}}/images/ubi/histogram.png "Vertical Bar Chart")
+
+## 6) Have fun slicing and dicing
+
+Now you can further filter the displayed data. For example, let's see how the click position changes when there is a purchase. Select **Add filter** and then select the `action_name:product_purchase` field, as shown in the following image.
+![Product Purchase]({{site.url}}{{site.baseurl}}/images/ubi/product_purchase.png "Product Purchase")
+
+
+You can filter event messages containing the word "\*laptop\*" by adding wildcards, as shown in the following image.
```

Comment:
```suggestion
You can filter event messages containing the word *laptop* by adding wildcards, as shown in the following image.
```

Reply:
The asterisks are part of the query.  I'll change to code formatting to help make that clearer